konami/iidx/product.md

```diff
@@ -0,0 +1,180 @@
+# Beatmania IIDX products
+
+TODO remove this once information is merged into respective products
+
+### Generation 1 cabinet
+
+| [KPI](../kpi.md) | Game      | Main PCB                  | IO                        | Monitor                                             |
+|------------------|-----------|---------------------------|---------------------------|-----------------------------------------------------|
+| `GQ863-JA`       | 1st Style | [Twinkle V1](#twinkle-v1) | [Twinkle IO](#twinkle-io) | [Toshiba rear projection](#toshiba-rear-projection) |
+| `GQ983-JA`       | Substream | [Twinkle V1](#twinkle-v1) | [Twinkle IO](#twinkle-io) | [Toshiba rear projection](#toshiba-rear-projection) |
+| `GQ985-JA`       | 2nd Style | [Twinkle V1](#twinkle-v1) | [Twinkle IO](#twinkle-io) | [Toshiba rear projection](#toshiba-rear-projection) |
+| `GQ992-JA`       | 3rd Style | [Twinkle V2](#twinkle-v2) | [Twinkle IO](#twinkle-io) | [Toshiba rear projection](#toshiba-rear-projection) |
+
+* Type of cabinet easily recognizable by grey colored front/coin door
+
+### Generation 2 cabinet
+
+| [KPI](../kpi.md) | Game        | Main PCB                              | IO          | Monitor               |
+|------------------|-------------|---------------------------------------|-------------|-----------------------|
+| `GQD01-JA`       | 10th Style  | [Bemani PC Type 1](#bemani-pc-type-1) | [D01](#d01) | [CRT](#CRT)           |
```

Comment:
Please, please, please don't use "BemaniPC Type X" terminology, but refer to the correct board in `boards.md`.